packager/react-packager/index.js

```diff
@@ -118,6 +117,16 @@ function createNonPersistentServer(options) {
   return createServer(options);
 }
 
+function omit(obj, blacklistedKeys) {
```

Comment:
Ideally, I'd like to replace `lodash` by a self-written functions (to eliminate one of the ext. dependencies). But I think it shouldn't approach it in this PR.